bookwyrm/static/css/bookwyrm.css

```diff
@@ -346,3 +346,33 @@ body {
         display: none;
     }
 }
+
+/* Book list
+ ******************************************************************************/
+
+ol.ordered-list {
+    list-style: none;
+    counter-reset: books;
```

Comment:
_books_ seems more specific than the name of the class implies. How about _items_ or having a more specific class name?